sections/node.zsh

```diff
@@ -42,6 +42,9 @@ spaceship_node() {
     return
   fi
 
+  # Get only the version
+  node_version=$(echo "$node_version" | grep -o 'v[0-9.]*' | head -n 1)
```

Comment:
Use `spaceship::grep` here. Refer to ansible, kotlin, php section for an example